src/renderers/shaders/ShaderChunk/skinning_pars_vertex.glsl.js

```diff
@@ -29,5 +29,68 @@ export default /* glsl */`
 
 	}
 
+	#ifdef DUAL_QUATERNION_SKINNING
+
+	vec4 getQuaternionFromMatrix( const in mat4 m ) {
```

Comment:
I know this would be substantially more work, but ideally you'd want to do the matrix conversion to dual quats on the javascript side and not in the shader as this is quite a heavy computation. So maybe an idea of future work. 

Though, I'll admit this way is less disruptive to the existing code and it's true Three.js is more about simplicity than efficiency.

Reply:
hi in mulitplyVectorWithDualQuaternion return should be 
return vec4(pos + tran, 1.0);
that solve translated 

but still miss bone scale tranform

Reply:
Thanks @lo-th for spotting the mistake 🎉 

I moved the calculations to the CPU and implemented scaling.